docs/sources/alerting/configure-notifications/manage-contact-points/_index.md

```diff
@@ -0,0 +1,147 @@
+---
+aliases:
+  - ../contact-points/ # /docs/grafana/<GRAFANA_VERSION>/alerting/contact-points/
+  - ../contact-points/create-contact-point/ # /docs/grafana/<GRAFANA_VERSION>/alerting/contact-points/create-contact-point/
+  - ../contact-points/delete-contact-point/ # /docs/grafana/<GRAFANA_VERSION>/alerting/contact-points/delete-contact-point/
+  - ../contact-points/edit-contact-point/ # /docs/grafana/<GRAFANA_VERSION>/alerting/contact-points/edit-contact-point/
+  - ../contact-points/test-contact-point/ # /docs/grafana/<GRAFANA_VERSION>/alerting/contact-points/test-contact-point/
+  - ../manage-notifications/manage-contact-points/ # /docs/grafana/<GRAFANA_VERSION>/alerting/manage-notifications/manage-contact-points/
+  - create-contact-point/ # /docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/create-contact-point/
+  - ./alerting-rules/
+  - ./alerting-rules/manage-notifications/manage-contact-points/
+  - alerting/manage-notifications/manage-contact-points/configure-integrations/
+  - ./alerting-rules/manage-contact-points/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/
+  - ./alerting-rules/create-notification-policy/ # /docs/grafana/latest/alerting/alerting-rules/create-notification-policy/
+  - ../alerting-rules/manage-contact-points/integrations/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/integrations/
+  - ../manage-notifications/manage-contact-points/ # /docs/grafana/latest/alerting/manage-notifications/manage-contact-points/
+  - ./alerting-rules/manage-contact-points/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/
+  - ./manage-contact-points/integrations/ # /docs/grafana/latest/alerting/alerting-rules/manage-contact-points/integrations/
```

Comment:
These look to be duplicates of https://github.com/grafana/grafana/pull/83372/files#diff-7a9297b42c747e453ee6c047efbf8e88377df6285ced3ec7b9dafa1681976043R13
and https://github.com/grafana/grafana/pull/83372/files#diff-7a9297b42c747e453ee6c047efbf8e88377df6285ced3ec7b9dafa1681976043R15
```suggestion
```